docs/source/guide/distributed-hetero.rst

```diff
@@ -0,0 +1,94 @@
+for `T0` and store it as node data of `T0`.
+
+.. code:: python
+
+    g.nodes['T0'].data['feat1'] = dgl.distributed.DistTensor((g.number_of_nodes('T0'), 1), th.float32, 'feat1',
+                                                             part_policy=g.get_node_partition_policy('T0'))
+
+The partition policies used for creating distributed tensors and embeddings are initialized when a heterogeneous
+graph is loaded into the graph server. A user cannot create a new partition policy at runtime. Therefore, a user
+can only create distributed tensors or embeddings for a node type or edge type.
+Accessing distributed tensors and embeddings also requires type-wise IDs.
+
+7.3.2 Distributed sampling
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+DGL v0.6 uses homogeneous IDs in distributed sampling. **Note**: this will change in the future release.
+DGL provides four APIs to convert node IDs and edge IDs between the homogeneous IDs and type-wise IDs: 
+
+* :func:`~dgl.distributed.GraphPartitionBook.map_to_per_ntype`: convert a homogeneous node ID to type-wise ID and node type ID.
+* :func:`~dgl.distributed.GraphPartitionBook.map_to_per_etype`: convert a homogeneous edge ID to type-wise ID and edge type ID.
+* :func:`~dgl.distributed.GraphPartitionBook.map_to_homo_nid`: convert type-wise ID and node type to a homogeneous node ID.
+* :func:`~dgl.distributed.GraphPartitionBook.map_to_homo_eid`: convert type-wise ID and edge type to a homogeneous edge ID.
+
+Below shows an example of sampling a subgraph with :func:`~dgl.distributed.sample_neighbors` from a heterogeneous graph
+with a node type called `paper`. We first convert type-wise node IDs to homogeneous node IDs. After sampling a subgraph
+from the seed nodes, we convert homogeneous node IDs and edge IDs to type-wise IDs and also store type IDs as node data
+or edge data.
+
+.. code:: python
+
+        gpb = g.get_partition_book()
+        # We need to map the type-wise node IDs to homogeneous IDs.
+        cur = gpb.map_to_homo_nid(seeds, 'paper')
+        # For a heterogeneous input graph, the returned frontier is stored in
+        # the homogeneous graph format.
+        frontier = dgl.distributed.sample_neighbors(g, cur, fanout, replace=False)
+        block = dgl.to_block(frontier, cur)
+        cur = block.srcdata[dgl.NID]
+
+        block.edata[dgl.EID] = frontier.edata[dgl.EID]
+        # Map the homogeneous edge Ids to their edge type.
+        block.edata[dgl.ETYPE], block.edata[dgl.EID] = gpb.map_to_per_etype(block.edata[dgl.EID])
+        # Map the homogeneous node Ids to their node types and per-type Ids.
+        block.srcdata[dgl.NTYPE], block.srcdata[dgl.NID] = gpb.map_to_per_ntype(block.srcdata[dgl.NID])
+        block.dstdata[dgl.NTYPE], block.dstdata[dgl.NID] = gpb.map_to_per_ntype(block.dstdata[dgl.NID])
+
+From node/edge type IDs, we can retrieve node/edge types. For example, `g.ntypes[node_type_id]`. Once we have
```

Comment:
To avoid "We" in this sentence, can this sentence become:
A user can retrieve node/edge type by using node/edge type IDS. ...... Once having node/edge type and type-wise IDs, a user can ......